pom.xml

```diff
@@ -3295,7 +3295,7 @@
                   com.puppycrawl.tools.checkstyle.checks.javadoc.AbstractJavadocCheckTest
                 </param>
                 <param>
-                  com.puppycrawl.tools.checkstyle.checks.AvoidEscapedUnicodeCharactersCheckTest
+                  com.puppycrawl.tools.checkstyle.checks.indentation.CommentsIndentationCheckTest
```

Comment:
`CommentsIndentationCheckTest` increased coverage to 100% in CodepointUtil and since this test also uses isBlank `AvoidEscape...check` is not needed. 
Can you suggest a comment?

Reply:
Please make a comment similar to https://github.com/checkstyle/checkstyle/blob/f5419e19e7a443ef31de2288aee9740b3f2de1d9/pom.xml#L3292

Reply:
done

Reply:
Resolved.